### PR TITLE
LPS-146828 Partial revert

### PR DESCRIPTION
**Pull request by victorg1991**
@Tim-Cao Should be fixed now :)

### CODE REVIEW

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by victorg1991** [comment]
ci:test:echo

**Comment by Tim-Cao** [comment]
Hi @victorg1991, would you mind having a look at integration test failures? Thanks

**Comment by Tim-Cao** [comment]
ci:test:bundle

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java [hunk 1]
```diff
@@ -234,22 +232,13 @@ public boolean containsLayoutRestrictedUpdatePermission(
...
-			}
-
-			return false;
-		}
+		if (contains(permissionChecker, layout, ActionKeys.UPDATE) ||
+			contains(
+				permissionChecker, layout, ActionKeys.UPDATE_LAYOUT_BASIC) ||
+			contains(
+				permissionChecker, layout, ActionKeys.UPDATE_LAYOUT_LIMITED) ||
+			contains(permissionChecker, layout, ActionKeys.UPDATE)) {
```
Comment (lfbesada) [inline portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java:240]: This condition is duplicating the first one

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/service/test/LayoutPermissionTest.java [hunk 2]
```diff
@@ -76,29 +75,19 @@ public void testContainsWithUpdateLayoutAdvancedOptionsPermission()
...
+		Assert.assertFalse(
+			_layoutPermission.containsLayoutUpdatePermission(
+				permissionChecker, layout));
+
+		Assert.assertFalse(
+			_layoutPermission.containsLayoutRestrictedUpdatePermission(
+				permissionChecker, layout));
+		Assert.assertFalse(
+			_layoutPermission.containsLayoutUpdatePermission(
+				permissionChecker, layout));
```
Comment (lfbesada) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/service/test/LayoutPermissionTest.java:90]: This should be removed, they are duplicating the above asserts

File: portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java [hunk 3]
```diff
@@ -270,25 +259,14 @@ public boolean containsLayoutUpdatePermission(
...
-		}
-
 		if (contains(permissionChecker, layout, ActionKeys.UPDATE) ||
+			contains(
+				permissionChecker, layout, ActionKeys.UPDATE_LAYOUT_BASIC) ||
+			contains(
+				permissionChecker, layout, ActionKeys.UPDATE_LAYOUT_CONTENT) ||
+			contains(
+				permissionChecker, layout, ActionKeys.UPDATE_LAYOUT_LIMITED) ||
+			contains(permissionChecker, layout, ActionKeys.UPDATE) ||
```
Comment (lfbesada) [inline portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java:269]: The same as above: this condition is duplicating the first one.

**Review by lfbesada — CHANGES_REQUESTED** [review]
Hey @victorg1991 LGTM 🤩 Just minor changes. Thanks!

File: portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java (hunk 3 above)
Comment (victorg1991) [reply portal-impl/src/com/liferay/portal/service/permission/LayoutPermissionImpl.java]: fixed!

**Comment by victorg1991** [comment]
All fixed @lfbesada Many thanks!!

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**